pkg/core/log.go

```diff
@@ -113,3 +113,69 @@ func HydrateLog(_type string, data string) (interface{}, error) {
 		panic("unknown type " + _type)
 	}
 }
+
+type Accounts map[string]Account
```

Comment:
I think this is confusing, if you just add an 's' to a name, we expect the type to be an array and not a Map.
Do you think we can use another name here ?

Reply:
don't have a better idea here :/